diff --git a/libdjvu/GException.cpp b/libdjvu/GException.cpp
index 4585ebe..41ba1e5 100644
--- a/libdjvu/GException.cpp
+++ b/libdjvu/GException.cpp
@@ -215,7 +215,7 @@ static void throw_memory_error() { G_THROW(GException::outofmemory); }
 static void (*old_handler)() = std::set_new_handler(throw_memory_error);
 # else 
 #   ifdef HAVE_STDINCLUDES
-static void (*old_handler)() = std::set_new_handler(throw_memory_error);
+//static void (*old_handler)() = std::set_new_handler(throw_memory_error);
 #   else
 static void (*old_handler)() = set_new_handler(throw_memory_error);
 #  endif // HAVE_STDINCLUDES
diff --git a/libdjvu/miniexp.cpp b/libdjvu/miniexp.cpp
index 1077cab..0cde359 100644
--- a/libdjvu/miniexp.cpp
+++ b/libdjvu/miniexp.cpp
@@ -1300,7 +1300,8 @@ static int stdio_fputs(miniexp_io_t *io, const char *s) {
 
 static int true_stdio_fgetc(miniexp_io_t *io) {
   FILE *f = (io->data[0]) ? (FILE*)(io->data[0]) : stdin;
-  return ::getc(f);
+//  return ::getc(f);
+  return getc(f);
 }
 static int compat_getc() { 
   return true_stdio_fgetc(&miniexp_io); 
